assignment_3/evaluation_file.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/bash
+
+if ! [ -f ~/se2001/assignment_3/script.sh ]; then
+	echo "File $HOME/se2001/assignment_3/script.sh not found" >&2
+	exit 1
+fi
+
+if ! [ -x ~/se2001/assignment_3/script.sh ]; then
+	echo "File $HOME/se2001/assignment_3/script.sh is not executable" >&2
+	exit 1
+fi
+
+num_keywords=`grep -o -e "echo" -e "define" -e "for" -e "return" -e "bc" script.sh | wc -l` 
```

Comment:
One can define the Fibonacci number without a for loop and return using recursion as well.

Reply:
We can remove this validation.

Reply:
Done, sir